### PR TITLE
per issue #238, add note about immutability

### PR DESCRIPTION
**Pull request by dhh1128**
See #238

***
<a href="https://pr-preview.s3.amazonaws.com/dhh1128/did-spec/pull/240.html" title="Last updated on Jul 19, 2019, 8:32 PM UTC (bcca013)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/did-spec/240/f3c7dae...dhh1128:bcca013.html" title="Last updated on Jul 19, 2019, 8:32 PM UTC (bcca013)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2487,6 +2487,48 @@ <h2>
...
+focused on a data model rather than a protocol, it offers no opinion about
+many aspects of how that model is put to use. However, individual DID methods
+may wish to consider constraints that would eliminate behaviors or semantics
+they don't need. The more "locked down" a DID method is, while providing the
+same set of features, the less it can be manipulated by malicious actors.
+        </p>
+        <p>
+As an example, consider the flexibility that the data model offers with respect
+to updating. A single edit to a DID doc can change anything and everything except
+the root <code>id</code> property of the document--and any individual JSON object
```
Comment (TallTed) [inline index.html]: `document--and`
-> `document &mdash; and`

File: index.html [hunk 2]
```diff
@@ -2487,6 +2487,48 @@ <h2>
...
+in the data model can change all of its properties except its <code>id</code>.
+But is it actually desirable for a service endpoint to change its <code>type</code>
+once it's been defined? Or for a key to change its value? Or would it be better
+to require a new <code>id</code> when certain fundamental properties of an
+object change? Malicious takeovers of a web site often aim for an outcome where the
+site keeps its identifier (the host name), but gets subtle, dangerous changes
+underneath. If certain properties of the site were required by spec to be immutable
+(e.g., the <a target="_blank" href="https://en.wikipedia.org/wiki/Autonomous_system_(Internet)">
+ASN</a> associated with its IP address), such attacks might be much harder and
+more expensive to carry out--and anomaly detection would be easier.
```
Comment (TallTed) [inline index.html]: `out--and`
-> `out &mdash; and`

File: index.html [hunk 3]
```diff
@@ -2487,6 +2487,48 @@ <h2>
...
 
+    <section>
+        <h2>
+Immutability
+        </h2>
+
+        <p>
+Many cybersecurity abuses hinge on exploiting gaps between reality and
+the assumptions of rational, good-faith actors. Like any ecosystem, the
+DID ecosystem has some potential for this to occur. Beccause this spec is
```
Comment (dlongley) [inline index.html]:
```suggestion
DID ecosystem has some potential for this to occur. Because this spec is
```

**Comment by msporny** [comment]
The group did also mention that the language should change (editorial approach, etc. should be fixed, but can be done after the merge).

**Comment by rhiaro** [comment]
Typos in this thread fixed in #244

**Pull request closed without merging**